frame/election-provider-multi-phase/src/benchmarking.rs

```diff
@@ -192,37 +190,22 @@ frame_benchmarking::benchmarks! {
 	}
 
 	on_initialize_open_signed {
-		// NOTE: this benchmark currently doesn't have any components because the length of a db
-		// read/write is not captured. Otherwise, it is quite influenced by how much data
-		// `T::ElectionDataProvider` is reading and passing on.
 		assert!(<MultiPhase<T>>::snapshot().is_none());
 		assert!(<MultiPhase<T>>::current_phase().is_off());
 	}: {
-		<MultiPhase<T>>::on_initialize_open_signed().unwrap();
+		<MultiPhase<T>>::on_initialize_open_signed();
```

Comment:
```suggestion
		<MultiPhase<T>>::on_initialize_open_signed()?;
```

Or dos this not return a result anymore?

Reply:
no longer a result.